README.md

```diff
@@ -2,7 +2,7 @@
 
 This repo hosts the `community.vmware` Ansible Collection.
 
-The collection includes the VMware modules and plugins supported by Ansible VMware community to help the management of VMware infrastructure.
+The collection includes the VMware modules and plugins supported by Ansible the VMware community to help the management of VMware infrastructure.
```

Comment:
disclaimer: I'm not a native english speaker.

'Ansible VMware commuity' is an (unformal) entity, I would keep that unchanged. But we can probably clarify with an extra sentense.